examples/gke-security-best-practices/gke-cluster.tf

```diff
@@ -0,0 +1,144 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+/*
+  This file manages GKE cluster resources
+*/
+
+//create our GKE cluster
+resource "google_container_cluster" "acme-cluster" {
```

Comment:
I'd suggest using the GKE module instead of directly creating the cluster: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine

Reply:
will address

Reply:
thank you kindly for the recommendation here! The background is that this demo will be used to deliver a workshop as we will refer back to the TF & GKE documentation. In an effort to keep things simple I'd prefer to use basic terraform for this iteration. I would like to consider the module for a future iteration of the project.